tutorials/layout/index.md

```diff
@@ -0,0 +1,1003 @@
+---
+layout: tutorial
+title: "Building Layouts in Flutter"
+sidebar: home_sidebar
+permalink: /tutorials/layout/
+---
+
+## Introduction
+
+This is a guide to building layouts in Flutter.
```

Comment:
I'm sure you're doing the right thing, but we say "layouts" and elsewhere: "lay out".